### PR TITLE
Avoid CuPy 7.1.0 for now

### PR DESCRIPTION
**Pull request by shwina**
Due to https://github.com/cupy/cupy/issues/2971

### CODE REVIEW

File: conda/environments/cudf_dev_cuda10.0.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - cupy>=6.6.0,<8.0.0a0
+  - cupy>=6.6.0,<7.1.0
```
Comment (jakirkham) [inline conda/environments/cudf_dev_cuda10.0.yml]: We could do `!=` just to skip this version. Seems likely they will fix it in the next release.

Comment (shwina) [reply conda/environments/cudf_dev_cuda10.0.yml]: So `cupy>=6.60,<8.0.0a0,!=7.1.0`?

Comment (jakirkham) [reply conda/environments/cudf_dev_cuda10.0.yml]: Yeah

Comment (shwina) [reply conda/environments/cudf_dev_cuda10.0.yml]: Thanks!

**Comment by jakirkham** [comment]
Thanks Ashwin! 😀

**Pull request merged**